res/gettext/naksu.pot

```diff
@@ -372,6 +398,13 @@ msgid ""
 "backup disk as exFAT."
 msgstr ""
 
+msgid "The server is already running."
+msgstr ""
+
+msgid ""
+"There is a server appears to be running but we remove it as you requested."
```

Comment:
Vähän jännältä kuulostaa. Ehkä jotain tyyliin "The server appears to be running --"?